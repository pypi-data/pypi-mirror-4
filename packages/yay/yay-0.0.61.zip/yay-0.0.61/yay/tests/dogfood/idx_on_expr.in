
lookup: a

in:
  - name: a
    action: fight
  - name: b
    action: flee
  - name: c
    action: nap

out: ${in[@.name = lookup][0]}

