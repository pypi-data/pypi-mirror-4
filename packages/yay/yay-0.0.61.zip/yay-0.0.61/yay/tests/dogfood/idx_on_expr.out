
lookup: a

in:
  - name: a
    action: fight
  - name: b
    action: flee
  - name: c
    action: nap

out:
  name: a
  action: fight

